src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -65,6 +65,8 @@ export default class MeetingContainer extends Component {
       teamMembers !== this.props.memberSub.data.teamMembers ||
       team.activeFacilitator !== oldTeam.activeFacilitator ||
       user.id !== this.props.user.id) {
+      // We've changed teams, reset our container's state:
+      this.resetContainer();
```

Comment:
oh, ew, yes, of course that doesn't work. Good eyes.

I believe we will need to catch when the teamId changes (as we can't always count on the container being cleanly umounted). I will rework this now.